src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureBase {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class declaration.